examples/models/llama2/README.md

```diff
@@ -91,6 +91,8 @@ If you want to deploy and run a smaller model for educational purposes. From `ex
     python -m examples.models.llama2.tokenizer.tokenizer -t tokenizer.model -o tokenizer.bin
     ```
 
+If you want to fine tune your model based on a specific dataset, PyTorch team provides a native-Pytorch library, [TorchTune](https://github.com/pytorch/torchtune), for easily authoring, fine-tuning and experimenting with LLMs.
+
```

Comment:
```suggestion

## (Optional) Finetuning

If you want to finetune your model based on a specific dataset, PyTorch team provides [TorchTune](https://github.com/pytorch/torchtune) - a native-Pytorch library for easily authoring, fine-tuning and experimenting with LLMs.
```

Reply:
Please feel free to disregard this suggestion.

Reply:
@kartikayk can you provide an example one-line-ish command to generate a new fine-tuned state_dict pth file?

Reply:
Oh sure, let me add this in the suggestion

Reply:
Done

Reply:
Thanks @kartikayk ! Will update.